src/main/java/book/store/repository/order/OrderRepository.java

```diff
@@ -0,0 +1,12 @@
+package book.store.repository.order;
+
+import book.store.model.Order;
+import java.util.List;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface OrderRepository extends JpaRepository<Order, Long> {
+    @Query("FROM Order o LEFT JOIN FETCH o.orderItems i WHERE o.user.id = :userId")
```

Comment:
it should work without custom queries
```suggestion
```

Reply:
<img width="1395" alt="Screenshot 2024-02-09 at 19 21 45" src="https://github.com/SosunovychB/online-book-store/assets/141525178/e64fb9cd-c5ed-4479-b482-af90f4be4f6a">
Unfortunately, it doesn't work.

Reply:
so it should be fixed by yourself. see my suggestion